godtools/App/Features/AppLanguage/Domain/Entities/AppLanguageNameDomainModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AppLanguageNameDomainModel.swift
+//  godtools
+//
+//  Created by Levi Eggert on 11/16/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+
+struct AppLanguageNameDomainModel {
+    
+    let name: String
+    
+    init(languageName: String, languageScriptName: String?) {
```

Comment:
Hey @rachaelblue here's another example where I was able to pass the raw data to the domain model and then include some business formatting rules to format the language name.  I think this will work.

![chinese-simplifed-and-chinese-traditional](https://github.com/CruGlobal/godtools-swift/assets/59846460/18b4cdc6-f2c5-44df-9112-a1d30b5ae25a)